macros/edr/materializations/tests/test.sql

```diff
@@ -1,3 +1,16 @@
+{% macro create_test_result_temp_table() %}
```

Comment:
Shall we maybe use [this macro](https://github.com/elementary-data/dbt-data-reliability/blob/6b8f5aa55b20743e99bf96e102b13dff2890e453/macros/utils/table_operations/create_temp_table.sql#L0-L1)?

Also, I'd clarify what `model["alias"]` means in the context of a test by exporting it to a variable.

Reply:
👍